apps/api/src/app/messages/usecases/get-messages/get-messages.usecase.ts

```diff
@@ -7,7 +7,11 @@ export class GetMessages {
   constructor(private messageRepository: MessageRepository, private subscriberRepository: SubscriberRepository) {}
 
   async execute(command: GetMessagesCommand) {
-    const LIMIT = 10;
+    const LIMIT = command.limit;
+
+    if (LIMIT > 1000) {
```

Comment:
Same here. I would store the default upper limit in a constant to not have to hardcode its value everywhere and also to prepare it to be configurable in the future.
Also nit lowercase the variable as it is not a constant in this method.